src/main/java/com/faforever/client/chat/KittehChatService.java

```diff
@@ -288,8 +290,10 @@ private void onChatUserQuit(UserQuitEvent event) {
 
   @Handler
   private void onTopicChange(ChannelTopicEvent event) {
-    Channel channel = event.getChannel();
-    getOrCreateChannel(channel.getName()).setTopic(event.getNewTopic().getValue().orElse(""));
+    String author = event.getNewTopic().getSetter().map(Actor::getName)
+        .map(name -> name.replaceFirst("!.*", "")).orElse("");
```

Comment:
This works, we could also check the type of the actor and use methods based on that but this is more general given it is part of the irc spec.